### DISCUSSION TITLE
如何在 vercel 部署中调整 Waline 参数？

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#930] [Q&A] [answered]
不太清楚前端语法操作方式，尝试在 `index.js` 中增加如下的代码：

```javascript

const Application = require('@waline/vercel');

module.exports = Application({
  wordLimit: '200', 
  uploadImage: false,
  async postSave(comment) {
    // do what ever you want after save comment
  }
});

```

然后在 vercel 上重新部署，发现 Waline Example 并没有变化，想请教一下熟悉前端的同学帮忙指教一下，谢谢。

**Comment by lizheming** [comment]
wordLimit 和 uploadImage 都是前端的配置，前端配置需要在你的博客网站中进行修改。

**Reply by elkan1788** [reply]
@lizheming 这层能理解的，这里的问题描述是 vercel 部署的那个示例，应该如何来调整这些 Waline 限制参数？

**Comment by elkan1788** [comment, marked as answer]
不知道还有没其他小伙伴知道怎么对在 Vercel 部署时的 Example 配置进行参数调整呢？ 还请熟悉前端开发的小伙伴帮忙看下，谢谢。

**Reply by lizheming** [reply]
Example 里面提供的前端页面只是用来做测试使用的，目前暂时不支持配置

**Comment by Mister-Hope** [comment]
你可以解释一下你需要编辑 Example 示例的动机是什么么？它也不是正常情况下需要使用的页面。而且任何前端配置都可以直接通过接口绕过。

**Reply by Mister-Hope** [reply]
如果你需要硬性限制，比如说评论的长度之类的，你可以在 Waline 提供的钩子中进行处理。

**Reply by lizheming** [reply]
已经在 issue 中处理了，无需跟进 https://github.com/walinejs/waline/issues/938
